LICENSE.md

```diff
@@ -0,0 +1,10 @@
+By accessing NServiceBus code here, you are agreeing to the following licensing terms.
```

Comment:
This doesn't look like the right content. It should be the from RepoStandards.

Reply:
I’ll copy pasted from another repo, to get it from RepoStandards can I simply copy it from that repo?

Reply:
Yes, that's fine.

Reply:
Copied from RepoStandards (https://github.com/Particular/NServiceBus.AmazonSQS/pull/308/commits/121a14891d2f0e2c42f0765d8860007fec3abb36)